cmd/influxd/run.go

```diff
@@ -249,6 +257,24 @@ func initializeServer(s *influxdb.Server, b *messaging.Broker) {
 	}
 }
 
+func openDefaultServer(s *influxdb.Server, b *messaging.Broker) {
+	// TODO: Create replica using the messaging client.
+
+	// Create replica on broker.
+	if err := b.CreateReplica(1); err != nil {
```

Comment:
what is the `1` being passed into `CreateReplica`? This is an entirely new cluster so `1` is the id of the server?